src/api/app/models/kiwi/repository.rb

```diff
@@ -23,7 +23,7 @@ class Kiwi::Repository < ApplicationRecord
   validates :order, numericality: { only_integer: true, greater_than_or_equal_to: 1 }
   # TODO: repo_type value depends on packagemanager element
   # https://doc.opensuse.org/projects/kiwi/doc/#sec.description.repository
-  validates :repo_type, inclusion: { in: %w(apt-deb rpm-dir rpm-md yast2) }
+  validates :repo_type, inclusion: { in: %w[apt-deb rpm-dir rpm-md yast2] }
```

Comment:
:question: when was this offense introduced? This validation was already there and in master Rubocop is working :thinking: